.gitignore

```diff
@@ -1,3 +1,5 @@
 libbpf/
 *.gitignore
 PKGBUILD
+target
+Cargo.lock
```

Comment:
Do we want these for reproducible builds?

Reply:
Yeah, I think we actually want to keep track of `Cargo.lock` changes in the commits.

Reply:
ahh ok, will add that in, I was trying to avoid unintended consequences of there being a top level Cargo.lock (but maybe there are none).